docs/organization.md

```diff
@@ -87,23 +87,23 @@ Get the current manager of a role.
 
 Set the manager of a role.
 
-| Name         | Type                        | Description                                                 |
-| ------------ | --------------------------- | ----------------------------------------------------------- |
-| `address`    | `String`                    | The entity to set as a role manager. Pass `null` to remove. |
-| `appAddress` | `String`                    | Address of the app instance.                                |
-| `roleId`     | `String`                    | The role identifier.                                        |
-| returns      | `Promise<TransactionPaths>` | A promise resolving to a `TransactionPaths` instance.       |
+| Name         | Type              | Description                                                 |
+| ------------ | ----------------- | ----------------------------------------------------------- |
+| `address`    | `String`          | The entity to set as a role manager. Pass `null` to remove. |
+| `appAddress` | `String`          | Address of the app instance.                                |
+| `roleId`     | `String`          | The role identifier.                                        |
+| returns      | `Promise<Intent>` | A promise resolving to an `Intent` instance.                |
 
 ### Organization#appIntent(appAddress, funcName, funcArgs)
 
 Execute a function on a given app.
 
-| Name         | Type                        | Description                                           |
-| ------------ | --------------------------- | ----------------------------------------------------- |
-| `appAddress` | `String`                    | Address of the app instance.                          |
-| `funcName`   | `String`                    | Name of the function to call.                         |
-| `funcArgs`   | `String`                    | Parameters to pass to the function.                   |
-| returns      | `Promise<TransactionPaths>` | A promise resolving to a `TransactionPaths` instance. |
+| Name         | Type              | Description                                  |
+| ------------ | ----------------- | -------------------------------------------- |
+| `appAddress` | `String`          | Address of the app instance.                 |
+| `funcName`   | `String`          | Name of the function to call.                |
+| `funcArgs`   | `String`          | Parameters to pass to the function.          |
+| returns      | `Promise<Intent>` | A promise resolving to an `Intent` instance. |
 
 ### Organization#appCall(appAddress, methodName, args)
```

Comment:
I wonder after the discussion on Notion with Brett and today talk with Dan from PieDao maybe we should think how we want to handle the `appCall` and `appState`. 

On one hand I think the `appCall` is a bit counter what we are trying to do with the graph. Maybe is a good way to have a fallback if some information is not accesable. On the other hand for the Ethereum connector might be dubplicating part of the code.

About the `appState` I wonder if we might want to lift this one and handle the `orgState`. I'm afraid if having the `appState` could make cross app awareness difficult as in the client.

Reply:
Another comment is on the `appEvent` I think the concept of an `appEvent` is related in particular with the Ethereum provider. I wonder how that would be represented on the connector for the graph for example.